### PR TITLE
open-cluster-management/backlog#4876 Add 2.0

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: install/upgrade_hub.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+[#upgrading-by-using-the-operator]
+= Upgrading by using the operator
+
+You control your {product-title} upgrades by using the operator subscription settings in the {ocp} console. When you initially deploy {product-title-short} by using the operator, you make the following selections:
+
+* Channel - Corresponds to the version of the product that you are installing. The initial channel setting is often the most current channel that was available at the time of installation. 
+
+* Approval - Specifies whether approval is required for updates within the channel, or if they are done automatically. If set to `Automatic`, then minor release updates in the selected channel are deployed without human intervention. If the `Manual` setting is selected, then each update to the minor release within the channel requires an administrator to approve the update.
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]: For consistency, I suggest that you make the following change:

...deployed without administrator intervention

let me know what you think

File: install/upgrade_hub.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+You control your {product-title} upgrades by using the operator subscription settings in the {ocp} console. When you initially deploy {product-title-short} by using the operator, you make the following selections:
+
+* Channel - Corresponds to the version of the product that you are installing. The initial channel setting is often the most current channel that was available at the time of installation. 
+
+* Approval - Specifies whether approval is required for updates within the channel, or if they are done automatically. If set to `Automatic`, then minor release updates in the selected channel are deployed without human intervention. If the `Manual` setting is selected, then each update to the minor release within the channel requires an administrator to approve the update. 
+
+You also use those settings when you upgrade {product-title-short} by using the operator. 
+
+*Required permissions:* {ocp-short} administrator
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]: Throughout the doc and in our meeting we agreed to use: **Required access**. I do like required permissions tho, maybe we can bring this up during the meeting

File: install/upgrade_hub.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+. Ensure that the _Upgrade Status_ is *Up to date*. This status indicates that the operator is at the latest level that is available in the selected channel. If the _Upgrade Status_ indicates that there is an upgrade pending, complete the following steps to update it to the latest minor release that is available in the channel:
+
+.. Click the *Manual* setting in the _Approval_ field to edit the value. 
+
+.. Select *Automatic* to enable automatic updates. 
+
+.. Select *Save* to commit your change. 
+
+.. Wait for the automatic updates to be applied to the operator. The updates automatically add the required updates to the latest version in the selected channel. When all of the updated updates are complete, the _Upgrade Status_ field indicates *Up to date*.
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]: Update When all of the updated updates are complete, the _Upgrade Status_ field indicates *Up to date*.

TO 

When all of the updates are complete, the _Upgrade Status_ field indicates *Up to date*.

File: install/upgrade_hub.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+. Log in to your {ocp-short} operator hub.
+
+. In the {ocp-short} navigation, select *Operators* > *Installed operators*.
+
+. Select the *{product-title}* operator.
+
+. Select the _Subscription_ tab to edit the subscription settings.
+
+. Ensure that the _Upgrade Status_ is *Up to date*. This status indicates that the operator is at the latest level that is available in the selected channel. If the _Upgrade Status_ indicates that there is an upgrade pending, complete the following steps to update it to the latest minor release that is available in the channel:
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]: Ensure that the _Upgrade Status_ indicates that it is *Up to date*.

or 

Ensure that the _Upgrade Status_ displays that it is *Up to date*.

or

Ensure that the _Upgrade Status_ is labeled  *Up to date*.

File: install/upgrade_hub.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+.. Select *Save* to commit your change. 
+
+.. Wait for the automatic updates to be applied to the operator. The updates automatically add the required updates to the latest version in the selected channel. When all of the updated updates are complete, the _Upgrade Status_ field indicates *Up to date*.
+
+. Now that the _Upgrade Status_ is *Up to date*, click the value in the _Channel_ field to edit it.  
+
+. Select the channel for the next available feature release. You cannot skip channels when upgrading. 
++
+*Important:* You cannot revert back to an earlier channel after upgrading to a later channel. You must uninstall the operator and reinstall it at an earlier version to use an earlier version.
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]: ...at an earlier version to use a previous version.

File: install/upgrade_hub.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+. Select *Save* to save your changes.
+
+. Wait for the automatic upgrade to complete. After the upgrade to the next feature release completes, the updates to the latest patch releases within the channel are deployed.
+
+. If you have to upgrade to a later feature release, repeat steps 7-9 until your operator is at the latest level of the desired channel. Make sure that all of the pathc releases are deployed for your final channel.
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]: should pathc be path?

**Review by dockerymick — APPROVED** [review]
Made a couple of comments, otherwise this looks good !

File: install/upgrade_hub.adoc (hunk 1 above)
Comment (cadawson) [reply install/upgrade_hub.adoc]: That works well. Rules out Brandi's cat from intervening. ;-) Thanks.

File: install/upgrade_hub.adoc (hunk 2 above)
Comment (cadawson) [reply install/upgrade_hub.adoc]: I can go with access.

File: install/upgrade_hub.adoc (hunk 3 above)
Comment (cadawson) [reply install/upgrade_hub.adoc]: You didn't like updated updates?

File: install/upgrade_hub.adoc (hunk 4 above)
Comment (cadawson) [reply install/upgrade_hub.adoc]: Done

File: install/upgrade_hub.adoc (hunk 5 above)
Comment (cadawson) [reply install/upgrade_hub.adoc]: Done

File: install/upgrade_hub.adoc (hunk 6 above)
Comment (cadawson) [reply install/upgrade_hub.adoc]: Nope, patch, but good catch.

File: install/upgrade_hub.adoc (hunk 5 above)
Comment (dockerymick) [reply install/upgrade_hub.adoc]: i take my suggestion back it is still repetitive lol. I made a commit and updated the statement to read: 

" You must uninstall the operator and reinstall an earlier version to use it."

Feel free to change it back if necessary

**Pull request merged**